tests/unit/test_simulated_stack.py

```diff
@@ -710,6 +724,89 @@ def testVariantAnnotationsSearch(self):
             fromJsonString(response.data)
         self.assertGreater(len(responseData.variantAnnotations), 0)
 
+    def testGetFeatureSet(self):
+        path = "/featuresets"
+        for dataset in self.dataRepo.getDatasets():
+            for featureSet in dataset.getFeatureSets():
+                responseObject = self.sendGetObject(
+                    path, featureSet.getId(), protocol.FeatureSet)
+                self.verifyFeatureSetsEqual(responseObject, featureSet)
+            for badId in self.getBadIds():
+                featureSet = features.AbstractFeatureSet(dataset, badId)
+                self.verifyGetMethodFails(path, featureSet.getId())
+        for badId in self.getBadIds():
+            self.verifyGetMethodFails(path, badId)
+
+    def testFeatureSetsSearch(self):
+        path = '/featuresets/search'
+        for dataset in self.dataRepo.getDatasets():
+            featureSets = dataset.getFeatureSets()
+            request = protocol.SearchFeatureSetsRequest()
+            request.datasetId = dataset.getId()
+            self.verifySearchMethod(
+                request, path, protocol.SearchFeatureSetsResponse, featureSets,
+                self.verifyFeatureSetsEqual)
+        for badId in self.getBadIds():
+            request = protocol.SearchFeatureSetsRequest()
+            request.datasetId = badId
+            self.verifySearchMethodFails(request, path)
+
+    def testGetFeature(self):
+        dataset = self.dataRepo.getDatasets()[0]
+        featureSet = dataset.getFeatureSets()[0]
+        request = protocol.SearchFeaturesRequest()
+        request.featureSetId = featureSet.getId()
+        request.referenceName = "chr1"
+        request.start = 0
+        request.end = 2**16
+        request.parentId = None
+        request.ontologyTerms = []
```

Comment:
Yes it should be. I had simply missed one occurrence in my `s/featureTypes/ontologyTerms/g`. Bless Python and its infinitely liberal attitude towards object attributes.